### PR TITLE
Ports VG's antimatter engine shield sprites and adds the AM engine to cargo

### PR DESCRIPTION
**Pull request by Shadowlight213**
old sprites:
amold
new sprites:
am
closes #20717

:cl:
add: Cargo can order the antimatter engine
/:cl:
I didn't decide to port the control sprite because I find the old one seems to fit better with the shields.
I'm going to work on making the UI be in a later PR. I just wanted to get this be used atm.

### CODE REVIEW

**Comment by WJohn** [comment]
Well these are a significant upgrade over... grey blobs. But they're still not great. I'm likely going to be running a pass over these at some point.

**Comment by Cheridan** [comment]
what's the design goal of this new engine

**Comment by Shadowlight213** [comment]
@Cheridan the AM engine can make extreme amounts of power. The plan is, once @Iamgoofball does goofpower, we will have it be essentially a high tier engine which would be used to make power for the Power transfer laser.
Antimatter would need to have its base components created from engineering, sent to another department for refining, and then sent to science for final creation and back to engineering for use.

**Comment by Supermichael777** [comment]
Got a slightly less awful version of that controller 
https://drive.google.com/open?id=0ByUBHqDNhfNdU2lKS1BReE1wSXc

less outline

**Comment by Cheridan** [comment]
@Shadowlight213 oh ok 
so its useless then

since singulo/tesla already makes way way more power than is actually needed and doesn't require you to go through any hoops

neat

**Comment by Shadowlight213** [comment]
@Cheridan Goofpower is going to nerf the tesla and sing, plus overhaul the power system in general.
Alternatively someone suggested using it to end up creating antimatter bombs.

**Comment by CyberbossJHCB** [comment]
We're getting a PTL? What's the reward? Cargo points?

**Comment by Shadowlight213** [comment]
@cyberboss yes. Eventually.
@Iamgoofball said it would be part of goofpower

**Comment by Anonmare** [comment]
I don't like the control panel's sprite, it looks very out of place compared to the rest of the engine and our existing sprites.

**Comment by KorPhaeron** [comment]
>the guy who added the infinite credit stock market, tier 4 upgrades, recylcers that turned nerf darts into stacks of diamonds, and the tesla that produced 1000 times as much power as the singulo is going to be the one to sit down and do the math to standardize engines and power drain on all the maps

Good one

**Comment by Iamgoofball** [comment]
>the guy who made a large quantity of fun features is making another fun feature

**Comment by ExcessiveUseOfCobblestone** [comment]
GOOF CAN DO NO WRONG

File: code/modules/power/antimatter/shielding.dm [hunk 1]
```diff
@@ -105,14 +107,37 @@
 
 
 /obj/machinery/am_shielding/update_icon()
+	dirs = 0
```
Comment (xxalpha) [inline code/modules/power/antimatter/shielding.dm]: Nooo no no, you have to make this use the smoothing system.

Comment (Shadowlight213) [inline code/modules/power/antimatter/shielding.dm]: Uhh. I'll try.
No clue how it works though.

**Comment by Anonmare** [comment]
BEHEAD THOSE WHO INSULT GOOFBALL

**Comment by kevinz000** [comment]
tfw you shoot two .357s at it and it makes a 8 gib radius explosive lel

File: code/modules/power/antimatter/shielding.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/modules/power/antimatter/shielding.dm]: I'm not sure you can make it use the smoothing system.

**Pull request merged**